docs/en/core-services/Catalog.md

```diff
@@ -102,17 +102,28 @@ $ ${ALLUXIO_HOME}/bin/alluxio table attachdb --db alluxio_db hive \
     thrift://metastore_host:9083 default
 ```
 
-If you want specify config file for the UDB, please add `attachdb` option `-o catalog.db.config.file`.
-Each time you want to affect the config file, you can use `alluxio table sync`.
-
-If you want to write a configuration file for the UDB, please configure it like the following given example.
-It must contain an array `bypassSet`, the comma-separated table names will be bypassed.
-
-```json
-{
-  "bypassSet" : ["table1", "table2"]
-}
-```
+If you want to specify a config file for the UDB, please append an option `-o catalog.db.config.file` to
+`attachdb` command.
+After each time you made changes to the config file, you can use `alluxio table sync` to apply the changes.
+
+The configuration file is in JSON format, and can contain these configurations:
+
+1. Tables and partitions bypassing specification:
+
+    ```json
+    {
+      "bypass": {
+        "tables": [
+          "table1",
+          {"table": "table2", "partitions": ["table2_part1", "table2_part2"]}
```

Comment:
(nit) Different from the number of tables in the database, there can be tens of thousands of partitions in single tables. Can we also come up with an easy way for user to configure the bypass partitions, like bypass major part of partitions? We can do it in a follow-up pr.

Reply:
@HelloHorizon 
> come up with an easy way for user to configure the bypass partitions, like bypass major part of partitions

Yes, that's the plan.

@maobaolong Baolong, could you please shed some light on how we should handle partition specifications to make it practical for real use cases?

Reply:
With the bypass table and partition feature, we can use it to bypass some older tables or partitions and mount a small range of tables or partitions to alluxio.

It just like a rolling forward window which stand for a range of active valuable dataset(partitions), the window can move forward by a time unit, let's say it is a week, so we just want alluxio cache the current week partition, and bypass the older partitions, if some query need the older partition, it will read data from under filesystem directly rather than read it from alluxio.

Reply:
@maobaolong thanks for you input. I will come up with an implementation outline and discuss with you later. Resolve this for now.